lib/screens/about_page.dart

```diff
@@ -3,8 +3,15 @@ import 'package:flutter/material.dart';
 import 'package:flutter_svg/flutter_svg.dart';
 import 'package:smarthome/helper/theme_manager.dart';
 import 'package:url_launcher/url_launcher.dart';
+import 'package:github/github.dart';
+import 'package:version/version.dart';
 
 class AboutScreen extends StatelessWidget {
+  static final Version version = new Version(1, 1, 5);

```

Comment:
Don't use new in dart, the compiler will use new or const where it fits
```suggestion
  static final Version version = Version(1, 1, 5);

```